setup.py

```diff
@@ -40,7 +40,7 @@ def finalize_options(self):
                                    'mmit/core/solver.cpp',
                                    'mmit/core/piecewise_function.cpp',
                                    'mmit/core/coefficients.cpp'],
-                          extra_compile_args=["-std=c++11"] + os_compile_flags)
+                          extra_compile_args=["-std=c++0x"] + os_compile_flags)
```

Comment:
not sure if this works on your computer, but I had to make this change to get it to compile on mine (Ubuntu)